Source/SwiftLintBuiltInRules/Rules/Lint/BlanketDisableCommandRule.swift

```diff
@@ -144,8 +146,9 @@ struct BlanketDisableCommandRule: Rule, SourceKitFreeRule {
             }
 
             if let command = ruleIdentifierToCommandMap[disabledRuleIdentifier] {
-                let reason = "The disabled '\(disabledRuleIdentifier.stringRepresentation)' rule " +
-                             "should be re-enabled before the end of the file"
+                let reason = "Use 'next', 'this' or 'previous' to disable the " +
+                             "'\(disabledRuleIdentifier.stringRepresentation)' rule for a single line or " +
+                             "'swiftlint:enable' to re-enable the rule immediately after the violations to be ignored"
```

Comment:
Hm ... these messages are typically short in nature. They succinctly mention what's wrong and sometimes give advice for the fix. The latter is rare and that's okay. For the reasons behind a violation, examples and ideas to fix it, one can have a look into the full documentation.

That said, I think this message is too long.

Reply:
So I've managed to trim it back to 132 characters plus the rule name (from 165 before).

`"Use 'next', 'this' or 'previous' to disable the 'colon' rule for a single line or 'swiftlint:enable' to re-enable it as soon as possible"`

Which is long, but not the longest reason I could find.

Even shorter would be:

`"Use 'next', 'this' or 'previous' to disable the 'colon' rule for one line or 'swiftlint:enable' to re-enable it as soon as possible"`

Happy to go back and forth on this for a bit to get it right. I'm trying to avoid saying anything about the "rest of the file" or the "end of the file".

Reply:
I like these more. Here is another shorter one:

`"Use 'next', 'this' or 'previous' instead to disable the 'colon' rule once or re-enable it as soon as possible"`

I think it's mentioning the possibilities. Details can be taken from the description.